### PR TITLE
B lorient548

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 1]
```diff
@@ -1,12 +1,13 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 schematypens="http://relaxng.org/ns/structure/1.0"
-?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
    schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BLorient548" type="mss">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
                 <title>Qǝddāse or Missal
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:8]: I would just keep Qǝddāse

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 2]
```diff
@@ -31,72 +32,199 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
+<msItem xml:id="ms_i1.1">
+<locus from="7r"/>
+<title xml:lang="en" type="complete" ref="LIT1716Kidanz">Daily prayer</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.2">
+<locus from="11r"/>
+<title xml:lang="gez" type="complete" ref="LIT1716Kidanz">ሊጦን፡ ዘሰርክ፡</title>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:54]: Is it rather this one LIT1808Litonz?

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 3]
```diff
@@ -31,72 +32,199 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<title xml:lang="en" type="complete" ref="LIT1347Epipha">The anaphora of Epiphanius</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i3.9">
+<locus from="120v"/>
+<title xml:lang="en" type="incomplete" ref="LIT3120Anapho">The anaphora of John Chrysostom</title>
+<textLang mainLang="gez"/>
+<note>Imperfect at the end.</note>
+</msItem>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:141]: Are they not part of the Missal?

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 4]
```diff
@@ -31,72 +32,199 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                    
-                    
-                    
+
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>Leaves torn out after <locus target="#f.65">f. 65</locus> and <locus target="f.#69">f. 69</locus>.</p>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:154]: this information would fit the element `<condition>`

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 5]
```diff
@@ -31,72 +32,199 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is written in a good hand, of the <date>19th century</date>; leaves tornout after <locus from="65" to="69"/>; imperfect at the end.
                                 </desc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:174]: the information on the torn-out leaves is not suitable here.

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 6]
```diff
@@ -31,72 +32,199 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
+                        <decoDesc>
+                          <decoNote xml:id="d1">
+                          <locus target="#7r #31r"/>
+                          <desc>Few attempts at ornament.</desc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:184]: Please, add type of the decoration

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 7]
```diff
@@ -31,72 +32,199 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         <additions>
-                            
+
+
+<list>
+<item xml:id="a1">
+<locus from="2r"/>
+<desc>A quire of six leaves has been prefixed to the volume, containing an additional service of hymns and prayers.</desc>
+</item>
+
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:195]: That means it is a composite manuscript, so the whole description should be reworked

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient548.xml (hunk 3 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient548.xml:141]: I looked at other missals, so you're right of course, I've rearranged the numbering.

File: LondonBritishLibrary/orient/BLorient548.xml (hunk 4 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient548.xml:154]: Thank you, that's very useful. I hope it's OK the way I did it.

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 8]
```diff
@@ -31,72 +31,209 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                                        <width>171</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>8.875</height>
+                                        <width>6.75</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+                                <condition key="deficient"><desc>Leaves torn out after <locus target="#f.65">f. 65</locus>
+                                and <locus target="f.#69">f. 69</locus></desc>.
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:170]: f. in target is not needed

File: LondonBritishLibrary/orient/BLorient548.xml (hunk 5 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient548.xml:183]: I guess that the information on the torn out leaves is not really needed here.

File: LondonBritishLibrary/orient/BLorient548.xml (hunk 6 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient548.xml:193]: sorry, I can not see the type

**Review by DariaElagina — CHANGES_REQUESTED** [review]

**Comment by DariaElagina** [comment]
I guess that since the manuscript is a composite one, a restructuring of the description is needed.

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 9]
```diff
@@ -31,89 +32,255 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
+                                            </handDesc>
+                                            <decoDesc>
+                                              <decoNote type="ornamentation" xml:id="d1">
+                                              <locus target="#7r #31r"/>
+                                              <desc>Few attempts at ornament.</desc>
+                                              </decoNote>
+                                              </decoDesc>
+                                            <additions>
+
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:81]: Condition, layout, handnote and deco should be moved to the corresponding part (part 1 I guess).

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 10]
```diff
@@ -31,89 +32,255 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+          <date notBefore="1800" notAfter="1900"/>
+
+      </handNote>
+
+  </handDesc>
+
+<additions>
+<list>
+  <item xml:id="e1">
+  <locus target="6v"/>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:272]: #6v

File: LondonBritishLibrary/orient/BLorient548.xml [hunk 11]
```diff
@@ -131,21 +298,25 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                 <keywords>
                     <term key="ChristianLiterature"/>
                     <term key="Missal"/>
-
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient548.xml:134]: Also prayer

**Review by DariaElagina — CHANGES_REQUESTED** [review]

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**